acceptance/analyzer_test.go

```diff
@@ -206,13 +206,28 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 		})
 
 		when("the provided layers directory isn't writeable", func() {
+			var imageName, authConfig string
+			var err error
+			it.Before(func() {
+				imageName = authRegistry.RepoName("some-image")
+				authConfig, err = auth.BuildEnvVar(authn.DefaultKeychain, imageName)
+				h.AssertNil(t, err)
+			})
```

Comment:
I feel like there's some refactoring we could do to avoid having to repeat this for every test (like moving it to a parent context). But we don't have to do that as part of this PR.

Reply:
Could we just use "read only registry" for this particular test? I think any test that was using "some-image" (or another non-existent image) before could use the read only registry.